recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
     fn: cmocean-{{ version }}.tar.gz
     url: https://github.com/matplotlib/cmocean/archive/{{ version }}.tar.gz
-    sha256: 22e40329f5fd1c6d2136d7b32dfe988f3b8570a360b86bf9786efb2cb20c9aaa
+    sha256: 7f52753cd7cb2a8ccd532dadbce8db149088e27e3ad9b9f05ec9a30b6c12db4c
```

Comment:
Huh, that is the command I used. Weird.

On Thu, Jul 21, 2016 at 12:01 PM, Jonathan J. Helmus <
notifications@github.com> wrote:

> In recipe/meta.yaml
> <https://github.com/conda-forge/cmocean-feedstock/pull/2#discussion_r71766789>
> :
>
> > @@ -7,7 +7,7 @@ package:
> >  source:
> >      fn: cmocean-{{ version }}.tar.gz
> >      url: https://github.com/matplotlib/cmocean/archive/{{ version }}.tar.gz
> > -    sha256: 22e40329f5fd1c6d2136d7b32dfe988f3b8570a360b86bf9786efb2cb20c9aaa
> > +    sha256: 7f52753cd7cb2a8ccd532dadbce8db149088e27e3ad9b9f05ec9a30b6c12db4c
>
> I believe on a Mac the command is shasum -a 256 file.tar.gz. Note that
> shasum must be installed using a package manager like MacPorts or Homebrew.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/cmocean-feedstock/pull/2/files/80a767d09897aba71d984815767b9421485f62af#r71766789>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADU2BRyWrmrfNBQSlPa60IfzwFXeWvO2ks5qX8IigaJpZM4JSFl_>
> .
>



-- 
Kristen M. Thyng
Assistant Research Professor
Department of Oceanography
Texas A&M University
Eller O&M 607
http://kristenthyng.com